CMakeLists.txt

```diff
@@ -430,17 +433,18 @@ set(Impacto_Header
         src/games/chlcc/systemmenu.h
         src/games/chlcc/trophymenu.h
         src/games/chlcc/delusiontrigger.h
-
+        
```

Comment:
Stray whitespace changes looks like

Reply:
isn't this one fine? it separates chlcc and cc

Reply:
oh never mind added a tab